touchforms/backend/xformplayer.py

```diff
@@ -445,7 +445,7 @@ def _parse_repeat_juncture(self, event):
         ro = r.getRepeatOptions()
 
         event.update(get_caption(r))
-        event['main-header'] = ro.header
```

Comment:
ok, thanks for confirming - I didn't see anywhere in the smsforms code that expects `main-header` so I think we're ok